packages/experimental-identity-and-auth/src/HttpAuthSchemeProvider.ts

```diff
@@ -23,5 +23,5 @@ export interface HttpAuthSchemeParametersProvider<
   C extends object = object,
   T extends HttpAuthSchemeParameters = HttpAuthSchemeParameters
 > {
-  (config: C, context: HandlerExecutionContext): Promise<T>;
+  (config: C, context: HandlerExecutionContext, input: Record<string, unknown>): Promise<T>;
```

Comment:
what is input?

Reply:
Input is the operation input. I took the type from `getEndpointFromInstructions()` and `resolveParams` in `@smithy/middleware-endpoint`.

https://github.com/awslabs/smithy-typescript/blob/38e2831b853d0f6c205406d1521278e32cfa45ed/packages/middleware-endpoint/src/adaptors/getEndpointFromInstructions.ts#L31